public/locales/en/event.json

```diff
@@ -76,6 +76,11 @@
     "deleteDateFilter": "Poista rajaus"
   },
   "keywords": {
-    "labelKeywordLink": "Search events with keyword {{keyword}}"
+    "labelKeywordLink": "Search events with keyword {{keyword}}",
+    "enrollmentStatus": {
+      "externally_unknown": "Registrering på en annan sida",
+      "free_space_available": "Platser kvar",
+      "full": "Full"
+    }
```

Comment:
enrollmentStatus in en & sv, enrolmentStatus in fi,
enrolmentStatus would be in line with the British English use in the backend i.e. enrolment (British English) instead of enrollment (American English). Either way a single name should be used in en/sv/fi json files.